src/my_project/tagging/metadata_controller.py

```
@@ -303,37 +291,50 @@ def get_one_result(self) -> Optional[TagApplication]:
+        if matchcode == MatchCode.ONE_MATCH:
+            publisher_info = tagger.get_publisher_info(tagger.results[0].volume.id)
+            detailed_info = tagger.http.detail_get_request(tagger.results[0].id)
             return TagApplication(
-                tagger.results[0], publisher_info, API_KEY, self.filename
+                detailed_info,
+                publisher_info,
+                self.config_manager.config.comicvine.api_key,
+                self.filename,
             )
-        elif len(tagger.results) > 1:
+        elif matchcode == MatchCode.MULTIPLE_MATCHES:
```

Ruff (0.16.1)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)